luigi/scheduler.py

```diff
@@ -108,12 +108,12 @@ class scheduler(Config):
     worker_disconnect_delay = parameter.FloatParameter(default=60.0)
     state_path = parameter.Parameter(default='/var/lib/luigi-server/state.pickle')
 
-    # Jobs are disabled if we see more than disable_failures failures in disable_window seconds.
+    # Jobs are disabled if we see more than retry_count failures in disable_window seconds.
     # These disables last for disable_persist seconds.
     disable_window = parameter.IntParameter(default=3600,
                                             config_path=dict(section='scheduler', name='disable-window-seconds'))
-    disable_failures = parameter.IntParameter(default=999999999,
-                                              config_path=dict(section='scheduler', name='disable-num-failures'))
+    retry_count = parameter.IntParameter(default=999999999,
+                                         config_path=dict(section='scheduler', name='retry-count'))
```

Comment:
Code should be `name='disable_failures`` so we keep backward compatiblity. Docs should encourage usage of the newest name `retry_count`, so people dont get deprecation warnings.

Just add upon commits. There's not point in "rolling back". In the end I'll squash everything anyway as the history is a bit messy already. :)